pennylane/devices/default_tensor.py

```diff
@@ -151,28 +181,38 @@ class DefaultTensor(Device):
     This device is designed to simulate large-scale quantum circuits using tensor networks. For small circuits, other devices like ``default.qubit`` may be more suitable.
 
     The backend uses the ``quimb`` library to perform the tensor network operations, and different methods can be used to simulate the quantum circuit.
-    Currently, only the Matrix Product State (MPS) method is supported, based on ``quimb``'s ``CircuitMPS`` class.
+    The supported methods are Matrix Product State (MPS), based on the ``quimb``'s ``CircuitMPS`` class, and Tensor Network (TN), based on the ``quimb``'s ``Circuit`` class.
```

Comment:
```suggestion
    The supported methods are Matrix Product State (``"mps"``) and full tensor network contraction (``"tn"``).
```

Accidentally deleted my comment/suggestion instead of editing it.
I think the level of detail about `Circuit` and `CircuitMPS` classes are not important at this stage.